Ginger/GingerCoreNET/Run/GingerExecutionEngine.cs

```diff
@@ -4577,6 +4577,7 @@ public void StopRun()
                 mExecutedActivityWhenStopped = (Activity)CurrentBusinessFlow.CurrentActivity;
                 mExecutedActionWhenStopped = (Act)CurrentBusinessFlow.CurrentActivity?.Acts.CurrentItem;
                 mExecutedBusinessFlowWhenStopped = (BusinessFlow)CurrentBusinessFlow;
+                CurrentBusinessFlow.CurrentActivity.Status = eRunStatus.Stopped;
```

Comment:
we discussed to remove this part and handle it as part of driver load waiting time right ?